### PR TITLE
Pioneer 13 cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes new cal csvs for Pioneer 13 cruise:
WFP cal csvs - DOFSTK, FLORTK, PARADK

### CODE REVIEW

File: calibration/DOFSTK/CGINS-DOFSTK-00209__20190709.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-0209,CC_frequency_offset,-7.828016e+002,source_file DOFST-K_SBE43F_SN_0209_Calibration_2019-07-09.zip > 0209.cal and SBE 43F SOCAdjusted O0209 10Jul19.pdf
+43-0209,CC_oxygen_signal_slope,2.8301e-004,use adjusted Soc
+43-0209,CC_residual_temperature_correction_factor_a,-5.007934e-003,
+43-0209,CC_residual_temperature_correction_factor_b,2.276959e-004,
+43-0209,CC_residual_temperature_correction_factor_c,-3.348547e-006,
+43-0209,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (ksargentWHOI) [inline calibration/DOFSTK/CGINS-DOFSTK-00209__20190709.csv]: Looks good.
KLS And SNW

File: calibration/FLORTK/CGINS-FLORTK-00830__20190617.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+830,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_830_Calibration_2019-06-17.zip > BBFL2-830 Charsheet.pdf
+830,CC_dark_counts_cdom,47,
+830,CC_dark_counts_chlorophyll_a,50,
+830,CC_dark_counts_volume_scatter,47,
+830,CC_depolarization_ratio,0.039,
+830,CC_measurement_wavelength,700,
+830,CC_scale_factor_cdom,0.2016,
+830,CC_scale_factor_chlorophyll_a,0.0120,
+830,CC_scale_factor_volume_scatter,2.2170E-06,
+830,CC_scattering_angle,124,
```
Comment (ksargentWHOI) [inline calibration/FLORTK/CGINS-FLORTK-00830__20190617.csv]: Looks good.
KLS and SNW

File: calibration/PARADK/CGINS-PARADK-20525__20190624.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20525,CC_dark_offset,1.1,source_file PARAD-K_QSP2200_SN_20525_Calibration_2019-06-24.pdf
+20525,CC_scale_wet,1.02E-17,
```
Comment (ksargentWHOI) [inline calibration/PARADK/CGINS-PARADK-20525__20190624.csv]: Looks good.
KLS and SNW

File: calibration/PARADK/CGINS-PARADK-20437__20190624.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20437,CC_dark_offset,1.2,source_file PARAD-K_QSP2200_SN_20437_Calibration_2019-06-24.pdf
```
Comment (ksargentWHOI) [inline calibration/PARADK/CGINS-PARADK-20437__20190624.csv]: Looks good.
KLS and SNW

File: calibration/FLORTK/CGINS-FLORTK-01029__20190617.csv [hunk 5]
```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1029,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1029_Calibration_2019-06-17.zip > BBFL2-1029 Charsheet.pdf
+1029,CC_dark_counts_cdom,43,
+1029,CC_dark_counts_chlorophyll_a,46,
+1029,CC_dark_counts_volume_scatter,48,
+1029,CC_depolarization_ratio,0.039,
```
Comment (ksargentWHOI) [inline calibration/FLORTK/CGINS-FLORTK-01029__20190617.csv]: Looks good.

KLS and SNW

File: calibration/DOFSTK/CGINS-DOFSTK-02978__20190711.csv [hunk 6]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2978,CC_frequency_offset,-7.691666e+002,source_file DOFST-K_SBE43F_SN_2978_Calibration_2019-07-11.zip > 2978.cal and SBE 43F O2978 SOCAdjusted 11Jul19.pdf
+43-2978,CC_oxygen_signal_slope,2.2176e-004,use adjusted Soc
+43-2978,CC_residual_temperature_correction_factor_a,-5.360127e-003,
+43-2978,CC_residual_temperature_correction_factor_b,2.773047e-004,
+43-2978,CC_residual_temperature_correction_factor_c,-3.867184e-006,
```
Comment (ksargentWHOI) [inline calibration/DOFSTK/CGINS-DOFSTK-02978__20190711.csv]: Looks good.
KLS and SNW

File: calibration/DOFSTK/CGINS-DOFSTK-02499__20190614.csv [hunk 7]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2499,CC_frequency_offset,-8.365213e+002,source_file DOFST-K_SBE43F_SN_2499_Calibration_2019-06-14.zip > 2499.cal and SBE 43F O2499 SOCAdjusted 05Jul19.pdf
+43-2499,CC_oxygen_signal_slope,3.3675e-004,use adjusted Soc
+43-2499,CC_residual_temperature_correction_factor_a,-3.565070e-003,
+43-2499,CC_residual_temperature_correction_factor_b,1.881445e-004,
+43-2499,CC_residual_temperature_correction_factor_c,-2.988133e-006,
+43-2499,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (ksargentWHOI) [inline calibration/DOFSTK/CGINS-DOFSTK-02499__20190614.csv]: Looks good.
KLS and SNW

File: calibration/FLORTK/CGINS-FLORTK-01031__20190625.csv [hunk 8]
```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1031,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1031_Calibration_2019-06-25.zip > BBFL2-1031_CharSheet.pdf
+1031,CC_dark_counts_cdom,46,
+1031,CC_dark_counts_chlorophyll_a,48,
+1031,CC_dark_counts_volume_scatter,48,
```
Comment (ksargentWHOI) [inline calibration/FLORTK/CGINS-FLORTK-01031__20190625.csv]: Looks good.
KLS and SNW

File: calibration/PARADK/CGINS-PARADK-20447__20190624.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,0.9,source_file PARAD-K_QSP2200_SN_20447_Calibration_2019-06-24.pdf
```
Comment (ksargentWHOI) [inline calibration/PARADK/CGINS-PARADK-20447__20190624.csv]: Looks good.

KLS and SNW

File: calibration/PARADK/CGINS-PARADK-20513__20190624.csv [hunk 10]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20513,CC_dark_offset,1.4,source_file PARAD-K_QSP2200_SN_20513_Calibration_2019-06-24.pdf
+20513,CC_scale_wet,1.02E-17,
```
Comment (ksargentWHOI) [inline calibration/PARADK/CGINS-PARADK-20513__20190624.csv]: Looks good.
KLS and SNW

File: calibration/FLORTK/CGINS-FLORTK-01466__20190617.csv [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+1466,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1466_Calibration_2019-06-17.zip > BBFL2-1466 Charsheet.pdf
+1466,CC_dark_counts_cdom,47,
+1466,CC_dark_counts_chlorophyll_a,48,
+1466,CC_dark_counts_volume_scatter,49,
+1466,CC_depolarization_ratio,0.039,
+1466,CC_measurement_wavelength,700,
+1466,CC_scale_factor_cdom,0.0811,
+1466,CC_scale_factor_chlorophyll_a,0.0139,
+1466,CC_scale_factor_volume_scatter,3.387E-06,
+1466,CC_scattering_angle,124,
```
Comment (ksargentWHOI) [inline calibration/FLORTK/CGINS-FLORTK-01466__20190617.csv]: Looks good.
KLS and SNW

File: calibration/DOFSTK/CGINS-DOFSTK-03370__20190709.csv [hunk 12]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3370,CC_frequency_offset,-8.277713e+002,source_file DOFST-K_SBE43F_SN_3370_Calibration_2019-07-09.zip > 3370.cal and SBE 43F O3370 SOCAdjusted 10Jul19.pdf
+43-3370,CC_oxygen_signal_slope,3.2196e-004,use adjusted Soc
+43-3370,CC_residual_temperature_correction_factor_a,-5.009995e-003,
+43-3370,CC_residual_temperature_correction_factor_b,2.214015e-004,
+43-3370,CC_residual_temperature_correction_factor_c,-3.170532e-006,
+43-3370,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (ksargentWHOI) [inline calibration/DOFSTK/CGINS-DOFSTK-03370__20190709.csv]: Looks good.
KLS and SNW

File: calibration/DOFSTK/CGINS-DOFSTK-02708__20190614.csv [hunk 13]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2708,CC_frequency_offset,-7.878433e+002,source_file DOFST-K_SBE43F_SN_2708_Calibration_2019-06-14.zip > 2708.cal and SBE 43F O2708 SOCAdjusted 05Jul19.pdf
+43-2708,CC_oxygen_signal_slope,3.4442e-004,use adjusted Soc
+43-2708,CC_residual_temperature_correction_factor_a,-4.247485e-003,
+43-2708,CC_residual_temperature_correction_factor_b,2.040788e-004,
+43-2708,CC_residual_temperature_correction_factor_c,-3.288786e-006,
+43-2708,CC_residual_temperature_correction_factor_e,0.036,
```
Comment (ksargentWHOI) [inline calibration/DOFSTK/CGINS-DOFSTK-02708__20190614.csv]: Looks good.
KLS and SNW

File: calibration/FLORTK/CGINS-FLORTK-01117__20190625.csv [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+1117,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1117_Calibration_2019-06-25.zip > BBFL2-1117_CharSheet.pdf
+1117,CC_dark_counts_cdom,47,
+1117,CC_dark_counts_chlorophyll_a,49,
+1117,CC_dark_counts_volume_scatter,49,
+1117,CC_depolarization_ratio,0.039,
+1117,CC_measurement_wavelength,700,
+1117,CC_scale_factor_cdom,0.0553,
+1117,CC_scale_factor_chlorophyll_a,0.0114,
+1117,CC_scale_factor_volume_scatter,3.240E-06,
+1117,CC_scattering_angle,124,
```
Comment (ksargentWHOI) [inline calibration/FLORTK/CGINS-FLORTK-01117__20190625.csv]: Looks good.
KLS and SNW

File: calibration/PARADK/CGINS-PARADK-20464__20190624.csv [hunk 15]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20464,CC_dark_offset,1.2,source_file PARAD-K_QSP2200_SN_20464_Calibration_2019-06-24.pdf
+20464,CC_scale_wet,8.66E-18,
```
Comment (ksargentWHOI) [inline calibration/PARADK/CGINS-PARADK-20464__20190624.csv]: Looks good.
KLS and SNW

File: calibration/DOFSTK/CGINS-DOFSTK-03506__20190612.csv [hunk 16]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3506,CC_frequency_offset,-8.616855e+002,source_file DOFST-K_SBE43F_SN_3506_Calibration_2019-06-12.zip > 3506.cal
+43-3506,CC_oxygen_signal_slope,3.088623e-004,no Soc adjust
+43-3506,CC_residual_temperature_correction_factor_a,-3.029136e-003,
+43-3506,CC_residual_temperature_correction_factor_b,1.805157e-004,
+43-3506,CC_residual_temperature_correction_factor_c,-2.780575e-006,
+43-3506,CC_residual_temperature_correction_factor_e,3.60E-02,
```
Comment (ksargentWHOI) [inline calibration/DOFSTK/CGINS-DOFSTK-03506__20190612.csv]: Looks good.
KLS and SNW

File: calibration/FLORTK/CGINS-FLORTK-01537__20190617.csv [hunk 17]
```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1537,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1537_Calibration_2019-06-17.zip > BBFL2-1537 Charsheet.pdf
+1537,CC_dark_counts_cdom,49,
+1537,CC_dark_counts_chlorophyll_a,48,
+1537,CC_dark_counts_volume_scatter,48,
+1537,CC_depolarization_ratio,0.039,
+1537,CC_measurement_wavelength,700,
+1537,CC_scale_factor_cdom,0.0647,
+1537,CC_scale_factor_chlorophyll_a,0.0121,
```
Comment (ksargentWHOI) [inline calibration/FLORTK/CGINS-FLORTK-01537__20190617.csv]: vendor pdf says chl a scale factor = 0.0143, rest are all good.
KLS and SNW

File: calibration/PARADK/CGINS-PARADK-20541__20190624.csv [hunk 18]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20541,CC_dark_offset,0.9,source_file PARAD-K_QSP2200_SN_20541_Calibration_2019-06-24.pdf
+20541,CC_scale_wet,1.04E-17,
```
Comment (ksargentWHOI) [inline calibration/PARADK/CGINS-PARADK-20541__20190624.csv]: Looks good.
KLS and SNW

File: calibration/PARADK/CGINS-PARADK-20542__20190624.csv [hunk 19]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20542,CC_dark_offset,0.9,source_file PARAD-K_QSP2200_SN_20542_Calibration_2019-06-24.pdf
+20542,CC_scale_wet,1.10E-17,
```
Comment (ksargentWHOI) [inline calibration/PARADK/CGINS-PARADK-20542__20190624.csv]: Looks good.
KLS and SNW

File: calibration/FLORTK/CGINS-FLORTK-01538__20190625.csv [hunk 20]
```diff
@@ -0,0 +1,11 @@
...
+1538,CC_angular_resolution,1.076,source_file 14300-02\FLORT-K_BBFL2_SN_1538_Calibration_2019-06-25.zip > BBFL2-1538_CharSheet.pdf
+1538,CC_dark_counts_cdom,47,
+1538,CC_dark_counts_chlorophyll_a,47,
+1538,CC_dark_counts_volume_scatter,47,
+1538,CC_depolarization_ratio,0.039,
+1538,CC_measurement_wavelength,700,
+1538,CC_scale_factor_cdom,0.0859,
+1538,CC_scale_factor_chlorophyll_a,0.0122,
+1538,CC_scale_factor_volume_scatter,3.618E-06,
+1538,CC_scattering_angle,124,
```
Comment (ksargentWHOI) [inline calibration/FLORTK/CGINS-FLORTK-01538__20190625.csv]: Looks good.
KLS and SNW

File: calibration/DOFSTK/CGINS-DOFSTK-03523__20190618.csv [hunk 21]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3523,CC_frequency_offset,-8.175604e+002,source_file DOFST-K_SBE43F_SN_3523_Calibration_2019-06-18.zip > 3523.cal and SBE 43F O3523 SOCAdjusted 05Jul19.pdf
+43-3523,CC_oxygen_signal_slope,3.4489e-004,use adjusted Soc
+43-3523,CC_residual_temperature_correction_factor_a,-3.784564e-003,
+43-3523,CC_residual_temperature_correction_factor_b,1.698220e-004,
+43-3523,CC_residual_temperature_correction_factor_c,-2.914452e-006,
+43-3523,CC_residual_temperature_correction_factor_e,3.60E-02,
```
Comment (ksargentWHOI) [inline calibration/DOFSTK/CGINS-DOFSTK-03523__20190618.csv]: Looks good.
KLS and SNW

File: calibration/FLORTK/CGINS-FLORTK-01537__20190617.csv [hunk 22]
```diff
@@ -0,0 +1,11 @@
...
+1537,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1537_Calibration_2019-06-17.zip > BBFL2-1537 Charsheet.pdf
+1537,CC_dark_counts_cdom,49,
+1537,CC_dark_counts_chlorophyll_a,48,
+1537,CC_dark_counts_volume_scatter,48,
+1537,CC_depolarization_ratio,0.039,
+1537,CC_measurement_wavelength,700,
+1537,CC_scale_factor_cdom,0.0647,
+1537,CC_scale_factor_chlorophyll_a,0.0143,
+1537,CC_scale_factor_volume_scatter,3.609E-06,
+1537,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTK/CGINS-FLORTK-01537__20190617.csv]: confirmed that change to chlorophyll scale factor looks correct - RGT

**Pull request merged**